docs/Operators.md

```diff
@@ -5044,6 +5045,158 @@ expect(node, inputs=[x], outputs=[y], name='test_globalmaxpool_precomputed')
 </details>
 
 
+### <a name="Gradient"></a><a name="gradient">**Gradient**</a>
+
+  Gradient operator computes the partial derivatives of a specific tensor to
+  some other tensors. This operator is widely used in gradient-based training
+  algorithms. To illustrate its use, let's consider a computation graph,
+  
+  ```
+  X -----.
+         |
+         v
+  W --> Conv --> H --> Gemm --> Y
+                        ^
+                        |
+                        Z
+  ```
+  
+  , where W and Z are trainable tensors. Note that operators' attributes are
+  omitted for the sake of simplicity. Let dY/dW (dY/dZ) be the gradient of
+  Y with respect to W (Z). The user can compute gradient by inserting Gradient
+  operator to form
+  
+  ```
+  W --> Conv --> H --> Gemm --> Y
+  |      ^              ^
+  |      |              |
+  |      X              Z
+  |      |              |
+  |      |   .----------'
+  |      |   |  (X/W/Z is the 1st/2nd/3rd input of Gradient as shown in "xs")
+  |      v   v
+  '---> Gradient(xs=["X", "W", "Z"], y="Y") ---> dY/dX (1st output of Gradient)
+         |   |
+         |   '-----------------------------------> dY/dW (2nd output of Gradient)
+         |
+         '---------------------------------------> dY/dZ (3rd output of Gradient)
+  ```
+  
+  , where the content inside the braces of Gradient are attributes. The attribute
+  "xs" specifies the necessary inputs to compute the variable specified by
+  attribute "y". To compute gradient, the runtime should identify a sub-graph with
```

Comment:
This feels to me like an implementation detail, not a definition of what the Gradient Operator does.

Reply:
Removed. Thanks!